gen/third_party/rplidar_sdk-release-v1.12.0/sdk/sdk/src/rplidar_driver.cpp

```diff
@@ -1821,7 +1821,7 @@ u_result RPlidarDriverImplCommon::grabScanData(rplidar_response_measurement_node
 
 u_result RPlidarDriverImplCommon::grabScanDataHq(rplidar_response_measurement_node_hq_t * nodebuffer, size_t & count, _u32 timeout)
 {
-    switch (_dataEvt.wait(timeout))
+    switch (int(_dataEvt.wait(timeout)))
```

Comment:
I believe these should no longer need a cast here since the actual function signature was changed in event.h.

Reply:
Done.